test/clojure/core_test/conj.cljc

```diff
@@ -0,0 +1,33 @@
+(ns clojure.core-test.conj
+  (:require [clojure.test :as t :refer [deftest testing is are]]
+            [clojure.core-test.portability #?(:cljs :refer-macros :default :refer) [when-var-exists]]))
+
+(when-var-exists conj
+  (deftest test-conj
```

Comment:
I think we can have a test for adding to the front of an infinite seq. For example `(conj (range) -1))` is valid.